sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -559,6 +562,248 @@ def set_read_only(
         except binascii.Error:
             raise binascii.Error("Connection string secret has incorrect padding")
 
+    @distributed_trace
+    def begin_create_snapshot(
+        self,
+        name: str,
+        filters: List[Dict[str, str]],
+        *,
+        composition_type: Optional[str] = None,
+        retention_period: Optional[int] = None,
+        tags: Optional[Dict[str, str]] = None,
+        **kwargs
+    ) -> LROPoller[Snapshot]:
+        """Create a snapshot of the configuration settings.
+
+        :param name: The name of the snapshot to create.
+        :type name: str
+        :param filters: A list of filters used to filter the configuration settings by key field and label field
+            included in the snapshot. Each filter should be in format {"key": <value>, "label": <value>},
+            and key field value is required.
+        :type filters: list[dict[str, str]]
+        :keyword str composition_type: The composition type describes how the key-values within the
+            snapshot are composed. The 'all' composition type includes all key-values. The 'group_by_key'
+            composition type ensures there are no two key-values containing the same key. Known values are:
+            "all" and "group_by_key".
```

Comment:
Can we type this as `Literal['all', 'group_by_key']`

Reply:
Updated